module7_selenium/src/test/java/page/calculator/GoogleCloudStartPage.java

```diff
@@ -1,26 +1,29 @@
-package page;
+package page.calculator;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.FindBy;
+import page.AbstractPage;
 
-public class GoogleCloudStartPage { //TODO
+public class GoogleCloudStartPage extends AbstractPage {
     private static final String GOOGLE_CLOUD_URL = "https://cloud.google.com/ ";
     private static final String SEARCH_QUERY = "Google Cloud Platform Pricing Calculator";
-    private WebDriver driver;
 
-    @FindBy(name ="q")
-    WebElement searchButton;
+    public GoogleCloudStartPage(WebDriver driver) {
+        super(driver);
+    }
 
-    public GoogleCloudStartPage openSite(){
+    public GoogleCloudStartPage openPage() {
         driver.get(GOOGLE_CLOUD_URL);
         return this;
     }
 
     public PageWithSearchResult findInfoOnTheSite() {
```

Comment:
Why this name?
Should be something like:
`searchFor(String query)`
or
`findGoogleCloudPlatformPricingCalculator`